src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -387,14 +387,14 @@
     short-summary: Indication that CIFS protocol is allowed
   - name: --nfsv3
     short-summary: Indication that NFSv3 protocol is allowed
-  - name: --nfsv4
-    short-summary: Indication that NFSv4 protocol is allowed
+  - name: --nfsv41
```

Comment:
The same as above. Please call out your change in History.rst

Reply:
I think this is already present.